tests/10apidoc/13ui-auth.pl

```diff
@@ -21,11 +24,35 @@ sub wait_for_cas_request
    });
 }
 
+# Generate a ticket request from the client to the homeserver (and "validate"
+# it with the CAS server).
```

Comment:
it looks like this assumes the client already has the ticket (it's passed in `ticket`) and now it's submitting it.

Also we don't validate it with the CAS server, since we *are* the CAS server. 

```suggestion
# Generate a ticket-submission request from the client to the homeserver.
#
# Waits for the validation request from the homeserver, and returns the given response.
```

Reply:
Right, that's why validate was in quotes, but definitely not clear enough! Thanks for the improved wording. 👍